components/NavLink.js

```diff
@@ -0,0 +1,12 @@
+import Link from 'next/link';
+
+export default function NavLink({ linkLabel, linkUrl }) {
+  return (
+    <Link href={linkUrl}>
+      {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
+      <a className="navegation__item">
+        <li>{linkLabel}</li>
```

Comment:
Yo creo que la estructura debería de ser `li>Link` dado que al llamar al componente este se encuentra dentro de un ul y actualmente se crea algo como `ul>a>li` perdiendo el sentido como elemento de la lista

Reply:
I don't think it's necessary to have an anchor tag inside the Link component.